service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C21OrderEmailContentProvider.java

```diff
@@ -43,24 +48,34 @@ protected C21OrderEmailContentProvider(@Value("${ccd.ui.base.url}")String uiBase
         // Validation within our frontend ensures that the following data is present
         CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
         final String subjectLine = buildSubjectLine(caseData);
-        return ImmutableMap.of(
+        return Map.of(
             "subjectLine", subjectLine,
             "localAuthorityOrCafcass", localAuthorityNameLookupConfiguration.getLocalAuthorityName(localAuthorityCode),
             "hearingDetailsCallout", buildSubjectLineWithHearingBookingDateSuffix(subjectLine, caseData),
+            "linkToDocStore", mostRecentUploadedC21DocumentUrl(caseData.getC21Orders()),
             "reference", String.valueOf(caseDetails.getId()),
             "caseUrl", uiBaseUrl + "/case/" + JURISDICTION + "/" + CASE_TYPE + "/" + caseDetails.getId()
         );
     }
 
+    private String mostRecentUploadedC21DocumentUrl(final List<Element<C21Order>> c21Orders) {
+        return getLast(c21Orders.stream()
+            .filter(Objects::nonNull)
+            .map(Element::getValue)
+            .filter(Objects::nonNull)
+            .collect(toList()))
+            .getDocument().getBinaryUrl();
+    }
+
     private String buildSubjectLineWithHearingBookingDateSuffix(final String subjectLine, final CaseData caseData) {
         String hearingDate = "";
-        if (!ObjectUtils.isEmpty(caseData.getHearingDetails())) {
+        if (!isEmpty(caseData.getHearingDetails())) {
             hearingDate = " hearing " + dateFormatterService.formatLocalDateToString(
                 hearingBookingService.getMostUrgentHearingBooking(caseData.getHearingDetails()).getDate(),
                 FormatStyle.MEDIUM);
         }
         return Stream.of(subjectLine, hearingDate)
             .filter(StringUtils::isNotBlank)
-            .collect(joining(", "));
```

Comment:
Was this change intentional? I'd imagine we'd want to have a space between our collections no?

Reply:
yes it was as _hearingDate_ already had a space.